### ISSUE TITLE
Change mobile app for events link on login page

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#1222]
Required new text:

"Looking for our mobile app for events? Download from the App Store or Google Play"

Where these link to:
App Store: https://itunes.apple.com/us/app/convene-2017/id1300582679?mt=8
Google Play: https://play.google.com/store/apps/details?id=com.zerista.ellenmacarthurfoundationevents

**Comment by asimonw** [comment]
Top comment now has the final copy.

**Comment by asimonw** [comment]
@fannyBkane This is not exactly like it's done on the mockup (seems to be a mix of the old way and the new way, so we must've caused some confusion), but I can live with this. wdyt?

**Comment by fannyBkane** [comment]
@asimonw @Cleop Is it possible to turn Apple store and Google play into links, and remove the 3rd line?

**Comment by SimonLab** [comment]
Cleo's new PR will amend the look of the links:
![image](https://user-images.githubusercontent.com/6057298/34830326-72564554-f6db-11e7-967f-fee59627e3ff.png)
I'll put the issue in please-test again once the PR is merged

**Comment by asimonw** [comment]
This looks good now.
